.github/workflows/containers.yml

```
@@ -0,0 +1,18 @@
+name: "Build and Publish Release Containers"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: "Build Containers"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build App CI container
+        run: docker build -f docker/app_ci . --build-arg="extra_vars=--extra-vars ccf_ver=$GITHUB_REF"
```

We should tag these as both the current version and latest. I think that's something like this?

```suggestion
        run: docker build -f docker/app_ci . --build-arg="extra_vars=--extra-vars ccf_ver=$GITHUB_REF" -t  ccfciteam/ccf-app-ci:$GITHUB_REF -t ccfciteam/ccf-app-ci
```

---

For latest, it's not so clear, unless we always release the latest. Let's do it separately when this works.